packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -54,6 +55,14 @@ module.exports = ({ model, strapi }) => {
     return _.omit(values, excludedKeys);
   };
 
+  const isValidPrimaryKey = params => {
```

Comment:
It might have more sense to rename it as `hasValidPrimaryKey`, what do you think?

Reply:
Done